web/package.json

```diff
@@ -25,6 +25,7 @@
     "@babel/cli": "^7.16.0",
     "@babel/core": "^7.18.13",
     "@babel/eslint-parser": "^7.18.2",
+    "@babel/helper-get-function-arity": "^7.16.7",
```

Comment:
I think [this](https://github.com/replicatedhq/kots/pull/3643/files) removal causes this issue.

I'm not very familiar with our webpack / babel setup. I need to take some time to clean it up more.